pkgs/development/libraries/openssl/default.nix

```diff
@@ -63,7 +65,9 @@ let
         inherit hash;
       };
 
-      inherit patches;
+      patches =
+        # https://cygwin.com/cgit/cygwin-packages/openssl/plain/openssl-3.0.18-skip-dllmain-detach.patch?id=219272d762128451822755e80a61db5557428598
+        patches ++ lib.optional stdenv.hostPlatform.isCygwin ./openssl-3.0.18-skip-dllmain-detach.patch;
```

Comment:
if it was `inherit patches`, then I think this patch should just be added to the invocations of `common`?

Reply:
Yeah, that's probably better.  It doesn't apply to 1.1 anyway